src/Metadata/Property/PropertyMetadata.php

```diff
@@ -321,4 +323,29 @@ public function withChildInherited(string $childInherited) : self
 
         return $metadata;
     }
+
+    /**
+     * JSON-LD value type.
```

Comment:
We try to make the Metadata component as independent as possible of serialization formats. IMO the value type is OK (it can be used in various serialization formats) but the comment should be made more generic.